### DISCUSSION TITLE
コンテンツのタグ毎に RSS をご提供頂いてますが、全てのコンテンツを取得する RSS は同様にご提供されてますでしょうか？

### DISCUSSION THREAD

**Discussion by susumuf** [increments/qiita-discussions#564] [要望・アイデア・不具合]
### 解決したいこと
現在、コンテンツのタグ毎に RSS をご提供頂いてますが、全てのコンテンツを取得する RSS は同様にご提供されてますでしょうか？

RSS に基づいて Microsoft Power Automate を利用して Microsoft Teams に対して RSS に含まれるコンテンツ内のカテゴリ情報（タグ）ごとにチャネルへアーカイブする様にして利用してますが、タグを対象としての RSS だけですと、個別のタグ毎に Power Automate タスクを用意している事から非常に手間がかかるので難儀してます。
出来れば全てのコンテンツを対象として、カテゴリ（タグ）に応じた Teams のチャネルを動的に作るように考えてます。

'Q&A' に対する RSS も探してみたんですが内容なんですが・・・無いんでしょうか？

### 自分で試したこと
探しては見たんですが見当たらないので質問させて頂きます。

**Comment by tanaka-asami415** [comment]
@susumuf 
お問い合わせいただきありがとうございます🙏

> 全てのコンテンツを取得する RSS は同様にご提供されてますでしょうか？

ご不便をおかけして申し訳ありません。現在全てのコンテンツを取得する RSS は提供できておりません🙇

時期は未定ですが今後フィードの改善も予定しているため、RSSの拡充も併せて検討したいと思います！

**Reply by susumuf** [reply]
前向きにご検討頂ければ幸いです。
